runtime/vm/jvminit.c

```diff
@@ -1074,7 +1074,7 @@ initializeJavaVM(void * osMainThread, J9JavaVM ** vmPtr, J9CreateJavaVMParams *c
 	} else if (j9sysinfo_processor_has_feature(&desc, J9PORT_X86_FEATURE_AVX)) {
 		vm->extendedRuntimeFlags |= J9_EXTENDED_RUNTIME_USE_VECTOR_REGISTERS;
 	}
-#endif /* defined(J9HAMMER) && (JAVA_SPEC_VERSION >= 17) */
+#endif /* defined(J9HAMMER) && (JAVA_SPEC_VERSION >= 17) && 0*/
```

Comment:
Missing space before end comment. Also please add a comment before the ifdef about this being disabled for performance reasons.